tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark.resource import cluster
+from rptest.clients.types import TopicSpec
+from rptest.services.admin import Admin
+
+import confluent_kafka as ck
+
+import random
+
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class TxAdminTest(RedpandaTest):
+    topics = (TopicSpec(name="tx_test",
+                        partition_count=1,
+                        replication_factor=3), )
+
+    def __init__(self, test_context):
+        super(TxAdminTest, self).__init__(test_context=test_context,
+                                          num_brokers=3,
+                                          extra_rp_conf={
+                                              "enable_idempotence": True,
+                                              "enable_transactions": True
+                                          })
+
+        self.admin = Admin(self.redpanda)
+
+    @cluster(num_nodes=3)
+    def test_simple_get_transaction(self):
+        self.producer1 = ck.Producer({
+            'bootstrap.servers':
+            self.redpanda.brokers(),
+            'transactional.id':
+            '0',
+        })
+        self.producer2 = ck.Producer({
+            'bootstrap.servers':
+            self.redpanda.brokers(),
+            'transactional.id':
+            '1',
+        })
+        self.producer1.init_transactions()
+        self.producer2.init_transactions()
+        self.producer1.begin_transaction()
+        self.producer2.begin_transaction()
```

Comment:
This test looks fine, but it would benefit from being generalized a bit so that it is easy to increase/decrease the scale. The simplest way to do this is in the test to introspect the `self.topics` list and produce to all topics then query all partitions for all topics. That way, you can easily change the number of topics and partitions without having to change any of the test code.